tests/test_host_check.py

```diff
@@ -2992,107 +2991,11 @@ def test_success(self, capsys):
             )
         assert textwrap.dedent(output) == textwrap.dedent(
             f"""\
-            PASS -- IOMMU
-                    IOMMU enabled for vfio-pci with the following PCI device(s):
-                    device1 (0000:00:00.0), device2 (0000:00:01.0), device3 (0000:00:02.0),
-                    device4 (0000:00:1f.2)
+            PASS -- IOMMU (IOMMU enabled for vfio-pci.)
             """
         )
         assert result is CheckState.SUCCESS
 
-    def test_no_iommu_devices(self, capsys):
```

Comment:
This test case is still relevant for the IOMMU check, I think - if there are no IOMMU devices (i.e., `iommu_devices = []`), the check should emit a warning (NEUTRAL if uio present, else WARNING).

Reply:
I don't think so, this test is actually for the case that no iommu pci devices are present, ie `iommu_devices & network_devices = []`, which is no longer part of `check_iommu`

Reply:
Is there another test case covering "no IOMMU devices"?